chia/full_node/mempool_check_conditions.py

```diff
@@ -49,6 +49,7 @@ def get_name_puzzle_conditions(
         if err is not None:
             return NPCResult(uint16(err), None, uint64(0))
         else:
+            assert result is not None
```

Comment:
I'm not sure why, but something made mypy realize that `result` is an `Optional` here. There's nothing in the type system describing that *either* `err` or `result` is set.